Source/GUI/Qt/GUI_Preferences.cpp

```diff
@@ -140,6 +140,7 @@ options Groups[Group_Max]=
             {"MD5_Verify", "Verify MD5 for audio data", Type_CheckBox, false},
             {"MD5_Embed", "Embed MD5 for audio data", Type_CheckBox, false},
             {"MD5_Embed_AuthorizeOverWritting", "Allow overwriting of embedded MD5 value", Type_CheckBox, false},
+            {"MD5_SwapEndian", "Swap endianness of MD5 representation", Type_CheckBox, false},

```

Comment:
Could you add an indicator on what is used? ie (unchecked is default is little endian (right?), checked is big endian)

Reply:
also perhaps to say "Display" is better than representation. Don't want to imply that this impacts storage.